packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/PdfViewer.tsx

```diff
@@ -21,6 +21,11 @@ interface Props {
    */
   scale: number;
 
+  /**
+   * Consider device pixel ration on rendering
+   */
+  useDeviceResolution?: boolean;
```

Comment:
Is there a reason for making this an option, rather than enabling it all the time?

Reply:
For the first time, I thought it's useful for debugging, but I can imagine no none would use the option. So I remove the option.